gemfirexd/shared/src/main/java/com/pivotal/gemfirexd/internal/shared/common/reference/SQLState.java

```diff
@@ -1713,6 +1713,7 @@ The message identifier (ie. the value that matches a field in this class)
     String LOB_OBJECT_LENGTH_UNKNOWN_YET = "XJ216.S";
     String LOB_LOCATOR_INVALID = "XJ217.S";
     String SNAPPY_OP_DISALLOWED_ON_COLUMN_TABLES = "XJ218.S";
+    String PUTINTO_OP_DISALLOWED_ON_COLUMN_TABLES = "XJ219.S";
```

Comment:
Is this error code still needed?

Reply:
No, it's not needed now. will remove it.